src/main/groovy/com/ullink/testtools/elastic/TestExportTask.groovy

```
@@ -137,7 +138,10 @@ class TestExportTask extends ConventionTask {
             }
         }
 
-        processor.close()
+        project.logger.info("Executing bulk export to Elasticsearch")
+        if (!processor.awaitClose(1, TimeUnit.HOURS)) {
```

Well, I think one hour is a bit long to start with. If I think about the usage of this plugin, a lot of projects will make us of it. If for some reason the ELK is down for instance, all jobs in CI will block during one hour, making the queue of jobs growing fast. IMO "fail fast(er)" is better. I'd suggest 10 minutes instead.
Isn't threre a way to wait and track progress instead (and make it fail when a certain amount of time is reached)? That would make more sense to me than a timeout like this one

---

It is true that one hour is a bit much to start with. I will reduce the default duration until we have a better idea of how much time is actually needed.

---

See #25